src/Uno.UI.Runtime.Skia.AppleUIKit/UI/Xaml/Controls/TextBox/InvisibleTextBoxViewExtension.cs

```diff
@@ -135,7 +139,7 @@ public void UpdateNativeView()
 	{
 		if (_owner.TextBox is { } textBox)
 		{
-			EnsureTextBoxView(textBox);
+			UpdateProperties();
 		}
```

Comment:
No need to assign anymore. Actually, no need to validate since `UpdateProperties()` already does a validation inside.

```suggestions
UpdateProperties();
```